src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/ndelius/Sentence.kt

```diff
@@ -1,5 +1,19 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.ndelius
 
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Term as IntegrationApiTerm
+
 data class Sentence(
   val date: String? = null,
-)
+  val length: Int? = null,
+  val lengthUnits: String? = null,
+) {
+  fun toTerm(): List<IntegrationApiTerm> {
+    return when (this.lengthUnits?.lowercase()) {
+      "years" -> listOf(IntegrationApiTerm(years = this.length))
+      "months" -> listOf(IntegrationApiTerm(months = this.length))
+      "weeks" -> listOf(IntegrationApiTerm(weeks = this.length))
+      "days" -> listOf(IntegrationApiTerm(days = this.length))
+      else -> listOf(IntegrationApiTerm())
```

Comment:
The example on their swagger spec suggests that "Hours" is also possible. How are going to manage this?
Currently it looks like it will fall in the "else" category, and meaning that we have changed the time of sentence from something in hours to something that's not populated.
https://external-api-and-delius-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html

Reply:
Further, what are all the possibilities of lengthUnits here? Can you link me to a source?

Reply:
Domain analysis yields the following from PreProd
`[
  {
    "domain": "COND_UNIT",
    "code": "DAYS",
    "description": "Days",
    "activeFlag": "Y",
    "listSeq": 2,
    "systemDataFlag": "Y",
    "subCodes": []
  },
  {
    "domain": "COND_UNIT",
    "code": "HOURS",
    "description": "Hours",
    "activeFlag": "Y",
    "listSeq": 1,
    "systemDataFlag": "Y",
    "subCodes": []
  },
  {
    "domain": "COND_UNIT",
    "code": "MONTHS",
    "description": "Months",
    "activeFlag": "Y",
    "listSeq": 4,
    "systemDataFlag": "Y",
    "subCodes": []
  },
  {
    "domain": "COND_UNIT",
    "code": "WEEKS",
    "description": "Weeks",
    "activeFlag": "Y",
    "listSeq": 3,
    "systemDataFlag": "Y",
    "subCodes": []
  },
  {
    "domain": "COND_UNIT",
    "code": "YEARS",
    "description": "Years",
    "activeFlag": "Y",
    "listSeq": 5,
    "systemDataFlag": "Y",
    "subCodes": []
  }
]`